dev/table_info.h

```diff
@@ -22,4 +22,24 @@ namespace sqlite_orm {
             name(move(name_)), type(move(type_)), notnull(notnull_), dflt_value(move(dflt_value_)), pk(pk_) {}
     };
 
+    struct table_xinfo {
```

Comment:
Why don't you simply extend `table_info` above?

Reply:
did not know if we were going to delete table_info now that it is obsolete!

Reply:
That doesn't matter. You just implement the way it should be. There's no point in splitting off an entire struct/class, where the old one is obsoleted completely anyway.

Reply:
but @fnc12 wants table_info as part of the public API!